about.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>AboutSection</title>
+    <link rel="stylesheet" href="./styles/aboutStyle.css">
+</head>
+<body>
+    <header class="header">
+        <a href="#" class="burger"><img src="./images/burgerButton.png" alt="Button Menu"></a>
+        <div class="nav-menu">
+            <div class="top-first d-flex">
+                <ul class="top-nav-ul d-flex">
+                    <li>
+                        <a href="https://www.instagram.com/alejandroramos_rasarunning/" target="_blank" rel="noopener">
+                            <img src="./images/instagramIcon.png" class="instagram" alt="Instagram link">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://twitter.com/AlejandroRBenji" target="_blank" rel="noopener">
+                            <img src="./images/twitterIcon.png" class="twitter" alt="Twitter link">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://linktr.ee/AlejandroRamos" target="_blank" rel="noopener" class="link-tree">Link Tree</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="top-second">
+                <div class="nav-main d-flex">
+                    <a href="index.html">
+                        <img src="./images/rasaLogo.png" alt="RasaRunning Logo">
+                    </a>
+                    <div class="nav-list">
+                        <ul class="d-flex">
+                            <li class="nav-list-about"><a href="about.html">About</a></li>
+                            <li>Races</li>
+                            <li>Blog</li>
+                            <li>Sponsors</li>
+                            <li>News</li>
+                            <li class="nav-list-major">Majors</li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
```

Comment:
- [ ] It would be better if you use a `nav` tag instead of a `div`. This will help you to organize your app and follow the HTML semantic tags. 
😄 💪